kfmon.c

```diff
@@ -1218,6 +1218,19 @@ static int
 	return 0;
 }
 
+static void
+	replace_nonchars(unsigned char *str, size_t length)
+{
+	for (size_t i = 0; i < length; i++)	{
+		unsigned char c = str[i];
+		if ((c >= 'a' && c <= 'z') || (c >= 'A' && c <= 'Z') || (c == ',') || (c == '-') || (c == '&') || (c == '?') || (c == 0) || (c >= '0' && c <= '9')) {
```

Comment:
That seems different than the list of illegal FAT32 chars. Was that from empirical evidence or just something to quickly start with?

Reply:
It sure helps when I actually read your first post, lol. Never mind me ;p.

Reply:
For reference, said list is `NUL, " * / : < > ? \ |` (according to https://en.wikipedia.org/wiki/Comparison_of_file_systems)